packages/service-library/src/global.d.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import 'jest-extended';
```

Comment:
why not make it global import so any test can use it or use import within test file itself instead of hiding it in d.ts file?